### PR TITLE
DEV: removes route action usage from sidebar footer

### PR DESCRIPTION
**Pull request by jjaffeux**
`routeAction` is an old and bad pattern we don't want to have in the codebase anymore.

### CODE REVIEW

File: app/assets/javascripts/discourse/app/components/sidebar/footer.gjs [hunk 1]
```diff
@@ -35,6 +36,16 @@ export default class SidebarFooter extends Component {
     this.modal.show(SidebarSectionForm);
   }
 
+  @action
+  showKeyboardShorcuts() {
```
Comment (CvX) [inline app/assets/javascripts/discourse/app/components/sidebar/footer.gjs:40]: 🩳 

```suggestion
  showKeyboardShortcuts() {
```

Comment (jjaffeux) [reply app/assets/javascripts/discourse/app/components/sidebar/footer.gjs]: weird I tried it, wondered how it works 🤔

**Review by CvX — APPROVED** [review]

File: app/assets/javascripts/discourse/app/components/sidebar/footer.gjs (hunk 1 above)
Comment (CvX) [reply app/assets/javascripts/discourse/app/components/sidebar/footer.gjs]: A pair of typos make it right! 😄

Comment (jjaffeux) [reply app/assets/javascripts/discourse/app/components/sidebar/footer.gjs]: ah yes ofc, for some reason I thought it was only one

**Pull request merged**